template.md

```diff
@@ -8,22 +8,36 @@ A brief, one or two sentence explanation of the proposal.
 
 # Example
 
-If the proposal involves a new or changed API, then;
+If the proposal involves a new or changed API, then include a basic code example.
+Otherwise, omit this section if it's not applicable.
 
-- include a basic code example; otherwise,
-- omit this section if it's not applicable.
+# Background
 
-# Motivation
+Include any useful background detail that that explains why this RFC is important.
+What situation created the problems and RFC goals outlined below? Why now?
+Be brief, if possible!
 
-In this section, step back and ask yourself:
+It can be useful to illustrate your RFC as a user problem in this section.
+(ex: "Users have reported that it is difficult to do X in Astro today.")
 
-> Why are we doing this? Why does it matter?
-> What use cases does it support? What is the expected outcome?
+# Goals
 
-Please focus on explaining the motivation, so that if this RFC is not accepted,
-the motivation could be used to develop alternative solutions. In other words,
-enumerate the constraints you are trying to solve without coupling them too
-closely to the solution you have in mind.
+A **bulleted-list** outlining the intended goals of this RFC. 
```

Comment:
Bulleted lists constrain you to one paragraph. What happens when a longer explanation is needed? I guess you can have a 1 sentence goal in the list and then expand on it in the same section.

Reply:
A bulleted list can still contain multi-sentence bullets if needed, and yes the "Background" section or even this section itself could contain more info to go into the details. 

But overall, it's an intentional move to push the author to summarize their goals as concisely as possible in a single place in the document.